packages/fxa-admin-panel/server/lib/client-config/index.ts

```diff
@@ -30,6 +32,13 @@ const defaultConfig: IClientConfig = {
       url: config.get('servers.admin.url'),
     },
   },
+  sentry: {
+    dsn: config.get('sentry.dsn'),
+    env: config.get('sentry.env'),
+    sampleRate: config.get('sentry.sampleRate'),
+    serverName: config.get('sentry.serverName'),
+    clientName: config.get('sentry.clientName'),
+  },
```

Comment:
is the reason we can't do `sentry = config.get('sentry')` here is that there are additional keys in the 'sentry' config?

Reply:
I actually did it this way since we set these properties explicitly in all of our other files as well. 🤔

Maybe we should check and change them all for consistency?